api/scripts/database/drop-database.js

```diff
@@ -11,9 +11,24 @@ const DB_TO_DELETE_NAME = url.pathname.slice(1);
 
 url.pathname = '/postgres';
 
+const databaseInstanceCanBeDropped = () => {
+  if (process.env.NODE_ENV === 'test') {
+    return true;
+  }
+  if (!process.env.ALLOW_DATABASE_DROP || process.env.ALLOW_DATABASE_DROP !== 'true') {
+    return false;
+  }
+  return true;
```

Comment:
Il me semble que ça revient à ça :
```suggestion
  return process.env.ALLOW_DATABASE_DROP === 'true';
```

Reply:
Corrigé